bin/run-clang-format.sh

```diff
@@ -29,6 +29,16 @@ else
 	else
 		echo "WARNING: installing the linter is not yet supported outside of Linux and Mac."
 	fi
+	
+	# Re-check after installation
+	if command -v clang-format-19 &> /dev/null; then
+		CLANG_FORMAT=clang-format-19
+	elif command -v clang-format &> /dev/null; then
```

Comment:
`clang-format` bin name different between mac and linux?

Reply:
yes https://github.com/viam-modules/orbbec/pull/8#discussion_r2183192207